docs/manual/scala/guide/cluster/Serialization.md

```diff
@@ -35,6 +30,18 @@ Having done that, you can provide the serializer registry by overriding the `jso
 If you need to use the registry outside of a Lagom application, for example, in tests, this can be done by customising the creation of the actor system, for example:
 
 @[create-actor-system](code/docs/home/scaladsl/serialization/Registry.scala)
+
+## Compression
+
+Compression, as described here, is only used for persistent events, persistent snapshots and remote messages with the service cluster. It is not used for messages that are serialized in the external API of the service.
+
+JSON can be rather verbose and for large messages it can be beneficial to enable compression. That is done by using the `JsonSerializable.compressed[T]` builder method instead of the `JsonSerializable.apply[T]` (as shown in the example snippet above):
```

Comment:
I think it's `JsonSerializer`, not `JsonSerializable`.

Moreover, the example uses a sugared apply version `JsonSerializer[T]`, so maybe better to have it in the docs instead of `JsonSerializer.apply[T]`